doc/userguide/devguide/extending/app-layer/overview.rst

```diff
@@ -0,0 +1,102 @@
+**************************
```

Comment:
We now have a heading order for all new documents: https://docs.suricata.io/en/latest/devguide/contributing/contribution-process.html#headings

Reply:
TIL